components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIAdminImpl.java

```diff
@@ -1018,4 +1034,26 @@ public void deleteTenantTheme(int tenantId) throws APIManagementException {
 
         apiMgtDAO.deleteTenantTheme(tenantId);
     }
+
+    @Override
+    public String getTenantConfig(String organization) throws APIManagementException {
+        return ServiceReferenceHolder.getInstance().getApimConfigService().getTenantConfig(organization);
+    }
+
+    @Override
+    public void updateTenantConfig(String organization, String config) throws APIManagementException {
+        if (schema != null) {
+            try {
+                org.json.JSONObject uploadedConfig = new org.json.JSONObject(config);
+                schema.validate(uploadedConfig);
+                APIUtil.validateRestAPIScopes(config);
+                ServiceReferenceHolder.getInstance().getApimConfigService().updateTenantConfig(organization, config);
+            } catch (ValidationException | JSONException e) {
+                throw new APIManagementException("tenant-config validation failure",
+                        ExceptionCodes.from(ExceptionCodes.INVALID_TENANT_CONFIG, e.getMessage()));
```

Comment:
should we omit e.getMessage() as otherwise it will send java error messages to client response?